helm/prometheus-operator/templates/cleanup.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    helm.sh/hook: pre-delete
+  labels:
+     app: {{ template "name" . }}
+     chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+     heritage: {{ .Release.Service }}
+     release: {{ .Release.Name }}
+  name: {{ template "fullname" . }}-cleanup-tpr
```

Comment:
Let's name this (and the pod template below) `{{ template "fullname" . }}-delete-tprs` to be consistent with the existing `{{ template "fullname" . }}-get-tprs` Job.

Also, rename the file to `delete-tprs-job.yaml`.